### DISCUSSION TITLE
Where is the source for https://duckdb.org/docs/guides/python/polars

### DISCUSSION THREAD

**Discussion by wgong** [duckdb/duckdb#6392] [General]
DuckDB is great, 
start reading the guide - https://duckdb.org/docs/guides/python/polars
On windows, executing the following command threw an error
```
pip install -U 'polars[pyarrow]'
ERROR: Invalid requirement: "'polars[pyarrow]'"
```
single quotes are not needed.
Where is the source to submit PR?
Thanks

**Comment by Mause** [comment]
The source code lives here: https://github.com/duckdb/duckdb-web/blob/master/docs/guides/python/polars.md

The quotes **are** required though, depending on your shell

**Reply by wgong** [reply]
Thanks, you are right, it is not required on Windows , but is on Linux, sorry about my inaccurate report

![image](https://user-images.githubusercontent.com/329928/220238916-eb821714-c570-4620-be1c-18d9e16c2940.png)
